packages/collapsible/test/lion-collapsible.test.js

```diff
@@ -0,0 +1,86 @@
+import { expect, fixture, html } from '@open-wc/testing';
+
+import '../lion-collapsible.js';
+
+const defaultCollapsible = html`
+  <lion-collapsible>
+    <button slot="invoker">More about cars</button>
+    <div slot="content">
+      Most definitions of cars say that they run primarily on roads, seat one to eight people, have
+      four tires, and mainly transport people rather than goods.
+    </div>
+  </lion-collapsible>
+`;
+
+describe('<lion-collapsible>', () => {
+  describe('Collapsible', () => {
+    it('sets expanded to false by default', async () => {
+      const collapsible = await fixture(defaultCollapsible);
+      expect(collapsible.expanded).to.equal(false);
+    });
+
+    it('has [expanded] on current expanded invoker which serves as styling hook', async () => {
+      const collapsible = await fixture(defaultCollapsible);
+      collapsible.expanded = true;
+      await collapsible.requestUpdate();
+      expect(collapsible).to.have.attribute('expanded');
+    });
+  });
+
+  /*
+  describe('User interaction', () => {
+    it('opens a invoker on click', async () => {
+      const collapsible = await fixture(defaultCollapsible);
+      const invoker = collapsible.querySelector('[slot=invoker]');
+      invoker.dispatchEvent(new Event('click'));
+      expect(collapsible.expanded).to.equal(true);
+    });
+  });
+  */
+
+  describe('Accessibility', () => {
+    it('is a11y AXE accessible', async () => {
+      const collapsible = await fixture(defaultCollapsible);
+      expect(collapsible).to.be.accessible();
```

Comment:
Can you also add a test for the expanded version?

Reply:
Added